pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.haml

```diff
@@ -28,7 +28,7 @@ theme: responsive_full_width
         %li Explore the curriculum and learning tools
         %li Experience the course as a teacher and a learner
         %li Collaborate with fellow teachers
-      %a{href: CDO.studio_url("/pd/application/teacher"), class: "link-button", target: "_blank", rel: "noopener noreferrer"} Apply now
+      %a{href: CDO.studio_url("/educate/professional-learning/program-information"), class: "link-button", target: "_blank", rel: "noopener noreferrer"} Apply now
```

Comment:
This part needs to be removed on these links: `CDO.studio_url(…)` since this page is on code.org and not studio.code.org.